clipmenu

```diff
@@ -69,3 +70,5 @@ done
 if (( CM_OUTPUT_CLIP )); then
     cat "$file"
 fi
+
+[ -n "$launcher_exit" ] && exit "$launcher_exit"
```

Comment:
Just write `exit "${launcher_exit:-"$?"}"` :-)

Reply:
Sure, that works too, changed it. :smile: